better-friends/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "better-friends",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@material-ui/core": "^3.9.3",
+    "axios": "^0.18.0",
+    "bootstrap": "^4.3.1",
+    "material-ui": "^0.20.2",
```

Comment:
are we wanting to use both bootstrap and material-ui ? could result in a large package. If not, maybe uninstall what we're not using